package.json

```diff
@@ -36,6 +36,7 @@
     "angular-schema-form-bootstrap": "^0.2.0",
     "angular-ui-bootstrap": "0.14.x",
     "bootstrap": "^3.3.6",
+    "bower": "^1.8.2",
```

Comment:
This should only be under `devDependencies`.

Reply:
done in f56c3e74a924c61f4d2df236f5d00915cb971dac